fence/agents/compute/fence_evacuate.py

```diff
@@ -0,0 +1,372 @@
+#!@PYTHON@ -tt
+
+import sys
+import time
+import atexit
+import logging
+import inspect
+import requests.exceptions
+
+sys.path.append("@FENCEAGENTSLIBDIR@")
+from fencing import *
+from fencing import fail_usage, is_executable, run_command, run_delay
+
+#BEGIN_VERSION_GENERATION
+RELEASE_VERSION="4.0.11"
+BUILD_DATE="(built Wed Nov 12 06:33:38 EST 2014)"
+REDHAT_COPYRIGHT="Copyright (C) Red Hat, Inc. 2004-2010 All rights reserved."
+#END_VERSION_GENERATION
```

Comment:
This section should be removed from both agents, as we've moved the essential part to lib/fencing.py to support reproducible builds.

https://github.com/ClusterLabs/fence-agents/pull/131/files#diff-94a84d829e5760b9a84e6d98bdba8fc4